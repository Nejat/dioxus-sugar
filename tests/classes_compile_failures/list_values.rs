use dioxus_sugar::classes;

#[classes(list(42, "42"))]
struct Props {}

fn main() {}