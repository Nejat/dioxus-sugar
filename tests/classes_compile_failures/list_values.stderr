error: lists are not supported
 --> tests/classes_compile_failures/list_values.rs:3:11
  |
3 | #[classes(list(42, "42"))]
  |           ^^^^^^^^^^^^^^
